tests/python/pants_test/base_test.py

```diff
@@ -437,6 +437,15 @@ def assertPrefixEqual(self, expected, actual_iter):
     """
     self.assertEqual(expected, list(itertools.islice(actual_iter, len(expected))))
 
+  def assertInFile(self, string, file):
+    """Verifies that a string appears in a file
+
+    :API: public
+    """
+
+    with open(file) as f:
+      self.assertIn(string, f.read())
```

Comment:
1. `file` is a built-in function in python, so it's good to rename it to something like `file_path` to avoid confusion.
2. Improve error msg 
```
content = f.read()
self.assertIn(string, content, '"{}" is not in the file {}:\n{}'.format(string, f.name, content)`